test-poc/framework/src/main/java/org/keycloak/test/framework/webdriver/WebDriverSupplier.java

```diff
@@ -0,0 +1,43 @@
+package org.keycloak.test.framework.webdriver;
+
+import org.keycloak.test.framework.injection.InstanceWrapper;
+import org.keycloak.test.framework.injection.LifeCycle;
+import org.keycloak.test.framework.injection.Registry;
+import org.keycloak.test.framework.injection.Supplier;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+public class WebDriverSupplier implements Supplier<WebDriver, InjectWebDriver> {
```

Comment:
Perhaps we should just call this FirefoxWebDriverSupplier? That would then go nicely together with https://github.com/orgs/keycloak/projects/42?pane=issue&itemId=63724099 as a way to select which supplier to use once we have multiple web driver suppliers implemented

Reply:
Yeah sure, can do. I just implemented as if it was already generic since we are missing the ability to select a specific instance at the moment.

Reply:
This was implemented under af0daed939459dc7f957bac985185ff6dd1df0d5